app/Options.hs

```diff
@@ -38,6 +39,10 @@ commandParser = subparser $
  <> command "remove"
             (info (helper <*> fmap Remove pathOptionsParser)
                   (fullDesc <> progDesc "Remove file or directory from the life configuration."))
+ <> command "push"
+            (info (helper <*> pure Push)
+                  (fullDesc <> progDesc "Updates GitHub repository with the life configuration."))
```

Comment:
I think it's better to tell something like `Updates GItHub repository from local state and push latest version`